playground/tailwind/vite.config.ts

```diff
@@ -27,11 +27,6 @@ export default defineConfig({
     // to make tests faster
     minify: false,
   },
-  server: {
-    // This option caused issues with HMR,
-    // although it should not affect the build
-    origin: 'http://localhost:8080',
-  },
```

Comment:
Technically this is testing `server.origin`, but this relies on rather redundant prepend/strip base manipulation, which is going to removed in https://github.com/vitejs/vite/pull/17886.